core/src/main/java/com/amplifyframework/auth/AuthCategoryBehavior.java

```diff
@@ -322,20 +336,6 @@ void confirmUserAttribute(
             @NonNull Consumer<AuthException> onError
     );
 
-    /**
-     * Update the password of an existing user - must be signed in to perform this action.
-     * @param oldPassword The user's existing password
-     * @param newPassword The new password desired on the user account
-     * @param onSuccess Success callback
-     * @param onError Error callback
-     */
-    void updatePassword(
-            @NonNull String oldPassword,
-            @NonNull String newPassword,
-            @NonNull Action onSuccess,
-            @NonNull Consumer<AuthException> onError
-    );
-
```

Comment:
Just out of curiosity what was the reason behind changing the order of this method?

Reply:
Previously, the series of user attribute methods were located between `confirmResetPassword()` and `updatePassword()`. Now I want to put the methods related to password operations together.